packages/did/src/DidDocumentExporter/DidContexts.ts

```diff
@@ -40,6 +48,22 @@ export const DID_CONTEXTS = {
       },
     ],
   },
+  [ASSET_DID_CONTEXT_URL]: {
+    '@context': {
+      '@protected': true,
+      id: '@id',
+      chain:
+        'https://github.com/KILTprotocol/spec-asset-did/blob/43457f9d1119bff1c3152e368ddd4e3bdb9558cb/README.md',
+      namespace:
+        'https://github.com/KILTprotocol/spec-asset-did/blob/43457f9d1119bff1c3152e368ddd4e3bdb9558cb/README.md',
+      reference:
+        'https://github.com/KILTprotocol/spec-asset-did/blob/43457f9d1119bff1c3152e368ddd4e3bdb9558cb/README.md',
+      asset:
+        'https://github.com/KILTprotocol/spec-asset-did/blob/43457f9d1119bff1c3152e368ddd4e3bdb9558cb/README.md',
+      identifier:
+        'https://github.com/KILTprotocol/spec-asset-did/blob/43457f9d1119bff1c3152e368ddd4e3bdb9558cb/README.md',
+    },
+  },
```

Comment:
This should have been discussed in the AssetDID specification then, I don't think this is the best place to talk about this. It's just implementation of something we have agreed on before. I am fine digging into the spec itself, and perhaps defining a top-level property that can include both chain and asset side. Probably the spec was rushed and not really checked by anyone else.

Reply:
I like your suggestion actually. And it's a very bad mistake on my side to not even try to use the JSON-LD playground, which is an amazing tool. I opened a draft PR in the spec repo, we can discuss there about what the best context definition is and what the best document for a resolved AssetDID is -> https://github.com/KILTprotocol/spec-asset-did/pull/1. We will come back to this PR afterwards. I think there is no issue in fixing the specification with breaking changes, since nobody relies on them anyway now. We only have a `parse` function in the SDK so far.